liquid-staking/src/common/storage.rs

```diff
@@ -0,0 +1,71 @@
+use multiversx_sc::imports::*;
+pub type Epoch = u64;
+
+#[multiversx_sc::module]
+pub trait CommonStorageModule {
+    #[view(getDelegationAddress)]
+    #[storage_mapper("delegationAddress")]
+    fn delegation_addresses(
+        &self,
+        contract_name: &ManagedBuffer,
+    ) -> SingleValueMapper<ManagedAddress>;
+
+    #[view(getDelegatedValue)]
+    #[storage_mapper("delegatedValue")]
+    fn delegated_value(&self, validator: &ManagedAddress) -> SingleValueMapper<BigUint<Self::Api>>;
```

Comment:
No need to specify `<Self::Api>` for framework types.

```suggestion
    fn delegated_value(&self, validator: &ManagedAddress) -> SingleValueMapper<BigUint>;
```